tests/test_trustme.py

```diff
@@ -359,3 +361,44 @@ def test_backcompat():
     ca = CA()
     # We can still use the old name
     ca.issue_server_cert(u"example.com")
+
+
+def test_CN():
```

Comment:
```suggestion
@pytest.fixture
def ca():
    return CA()


def test_CN(ca):
```

Reply:
If we're going to do this, we should do it for all the tests that start with `ca = CA()`. I have mixed feelings, though... a single function call is already about as abstracted-away as you can get. Putting it in a fixture barely simplifies anything, and adds the extra complication that now you need to know what this fixture does. I'm not like, categorically opposed, but I probably won't take the effort to set it up in this PR either.

Reply:
Well, it's more about doing lots of assertions of different flows in `test_CN` function. It's much better to see a lot of different items in test runner output instead of one meta item. Plus if some assert fails in the beginning, others don't get checked at all which is also harmful practice. That's where my suggestion to use fixture comes from. I wanted a set of small tests and `CA()` was one of the repeatable parts there.

Reply:
I can see the argument for large complex projects with lots of people working on them, but at the scale of a project like this I don't think it matters that much...